CMakeLists.txt

```diff
@@ -78,14 +78,14 @@ ENDIF()
 
 IF(EXISTS "${Trilinos_INCLUDE_DIRS}/KokkosCore_config.h")
```

Comment:
We can delete this if statement (and the else clause) and the `file(READ ...)` lines completely, I think, since we simply rely on CMake vars.